modules/context/context.go

```diff
@@ -353,7 +294,8 @@ func (ctx *Context) ServeContent(name string, r io.ReadSeeker, params ...interfa
 	http.ServeContent(ctx.Resp, ctx.Req, name, modtime, r)
 }
 
-// PlainText render content as plain text
+// PlainText render content as plain text, this will override basecontext's method
+// because we need a beautiful failed page
```

Comment:
I think you probably could drop this. We'll have already sent the header so the error page should only be rendered as text by the browser at the end.

Reply:
done.